queue_services/entity-emailer/src/entity_emailer/email_templates/CP-SR-CRCTN-COMPLETED.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base href="/">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="referrer" content="origin-when-cross-origin"/>
+    <meta name="author" content="BC Registries and Online Services">
+    <title>
+      Corrected documents from the BC Business Registry
+    </title>
+    [[style.html]]
+  </head>
+
+  <body>
+    <table class="body-table" role="presentation">
+      <tr>
+        <td>
+          [[header.html]]
+
+          <div class="container">
+            {% set sr_date = filing['resolutionDate'] %}
+            {% set filing_type = filing['correctedFilingType'] %}
+            <p>You have successfully filed a correction to the {{ sr_date }} {{ filing_type }} with the BC Business Registry.</p>
+
+            [[business-info.html]]
+            <p>The following documents are attached to this email:</p>
+            <ul class="outputs">
+                <li>Special Resolution</li>
+                {% if name_changed %}
+                    <li>Certificate of Name Change</li>
+                {% endif %}
+                {% if filing['rulesInResolution'] %}
+                    <li>Certified Rules</li>
+                {% endif %}
```

Comment:
I think `if filing['rulesInResolution'] %}` should be `if filing['rulesFileKey'] %}`.  "Certified Rules" should only show when a new rules pdf is uploaded right?

Reply:
you're right, I just updated